src/test/throttler/throttler.spec.ts

```diff
@@ -402,6 +402,28 @@ describe("Throttler", () => {
   });
 });
 
+describe("timeToWait", () => {
+  it("Should wait the base delay on the first attempt", () => {
+    const retryCount = 0;
+    const delay = 100;
+    const maxDelay = 1000;
+    expect(timeToWait(retryCount, delay, maxDelay)).to.equal(delay);
+  });
+  it("Should back off exponentially", () => {
```

Comment:
please have spaces between the cases (`it` blocks) and don't capitalize "Should")

Good thought to break out these tests though